src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -212,8 +214,7 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             response.Capabilities.Add("TypedDataCollection", bool.TrueString);
             response.Capabilities.Add("WorkerStatus", bool.TrueString);
             response.Capabilities.Add("HandlesWorkerTerminateMessage", bool.TrueString);
-            response.Capabilities.Add("HandlesInvocationCancelMessage", bool.TrueString);
-
+            response.Capabilities.Add("EnableUserCodeException", bool.TrueString);  //enableUserCodeExceptionOption.ToString());
```

Comment:
Can we remove the unused code comment here?

Reply:
Whoops had added that for testing - will change back. Good catch.